roles/evm/blockscout/handlers/main.yml

```diff
@@ -5,3 +5,5 @@
   service:
     name: blockscout
     state: restarted
+  when:
+  - skip_handlers | default("false") == "false"
```

Comment:
What kind of sorcery is that?

You can introduce the `blockscout_auto_restart` like in the [avalanche.node](https://github.com/AshAvalanche/ansible-avalanche-collection/blob/88b6bdf845f3c1c4399698fbe4afd56f120c8481/roles/node/handlers/main.yml#L8) role